visualization_scripts/style.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<vizmap id="VizMap-2019_11_22-16_53" documentVersion="3.0">
+    <visualStyle name="default">
+        <network>
+            <visualProperty default="1.0" name="NETWORK_SCALE_FACTOR"/>
+            <visualProperty default="400.0" name="NETWORK_HEIGHT"/>
+            <visualProperty default="0.0" name="NETWORK_CENTER_Z_LOCATION"/>
+            <visualProperty default="0.0" name="NETWORK_DEPTH"/>
+            <visualProperty default="0.0" name="NETWORK_CENTER_X_LOCATION"/>
+            <visualProperty default="true" name="NETWORK_EDGE_SELECTION"/>
+            <visualProperty default="false" name="NETWORK_ANNOTATION_SELECTION"/>
+            <visualProperty default="" name="NETWORK_TITLE"/>
+            <visualProperty default="#FFFFFF" name="NETWORK_BACKGROUND_PAINT"/>
+            <visualProperty default="550.0" name="NETWORK_SIZE"/>
+            <visualProperty default="550.0" name="NETWORK_WIDTH"/>
+            <visualProperty default="0.0" name="NETWORK_CENTER_Y_LOCATION"/>
+            <visualProperty default="true" name="NETWORK_NODE_SELECTION"/>
+        </network>
+        <node>
+            <dependency value="true" name="nodeCustomGraphicsSizeSync"/>
```

Comment:
Are these custom node graphics used in the initial visualization or can we simplify the style file?

Reply:
I created the `style.xml` within Cytoscape by dragging and dropping nodes and setting edge attributes and properties manually. Once I had a style that worked, I exported the style from Cytoscape using the export command. I have not done much editing of the actual file so this can be something we can look at to clean up.

Reply:
Okay, that sounds good.  In that case let's save manually simplifying the XML style file for a subsequent pull request, if it is needed at all.